src/MIDebugEngine/AD7.Impl/AD7Property.cs

```diff
@@ -414,6 +415,18 @@ public int SetValueAsStringWithError(string pszValue, uint dwRadix, uint dwTimeo
             }
             return Constants.E_FAIL;
         }
+
+        public int GetDataBreakpointInfo160(out string pbstrAddress, out uint pSize, out string pbstrDisplayName, out string pbstrError)
+        {
+            int hr = Constants.S_OK;
+
+            pbstrAddress = _variableInformation.Address();
+            pSize = _variableInformation.Size();
+            pbstrDisplayName = _variableInformation.Name; // or do we want FullName here?
+            pbstrError = "";
```

Comment:
try/catch and fill out the error string?

Reply:
For the catch clause, are the following assignments okay?
- pbstrAddress = string.Empty
- pSize = 0
- pbstrDisplayName = string.Empty
- pbstrError = e.Message where e is a generic Exception

Reply:
You might want to use null instead of string.Empty. But otherwise looks correct.